src/data/StorageTypes.ts

```diff
@@ -7,14 +7,21 @@ export type StorageAreaItemType = {
   quantity?: number;
 };
 
+export type StorageAreaBoxTemplate = {
+  boxTemplateId: string;
+  name: string;
```

Comment:
Thanks for showing me how to make it more complicated!

Reply:
:)